EveripediaNetwork/ep-api
# feat: added configuration for pm2 with service index & adjusted lint …

…for the Indexer

# Indexer Service running persistently 

_I have been setting up pm2 to keep the service running and testing it out locally. Some fixes were made in the Indexer Service so I could run it properly with pm2 now_

## How should this be tested?

1. Run pm2 start ecosystem.config.js
2. Check if the service keeps running

## Notes or observations

_I need to make it clear that it's running by showing off more logs. I have been having a hard time checking out if it's actually running our service and how it's working._

## Linked issues

closes #00, closes #001, closes https://github.com/EveripediaNetwork/issues/issues/122
